src/containers/App/App.js

```diff
@@ -1,39 +1,51 @@
 import './App.css';
-import { getDefaultData }from '../../ApiFetch'
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
-import { getAllMovies } from '../../actions'
-import Gallery from '../../components/Gallery/Gallery'
-import UserMenu from '../UserMenu/UserMenu'
-export class App extends Component {
+import { getDefaultData } from '../../ApiFetch';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getAllMovies } from '../../actions';
+import Gallery from '../../components/Gallery/Gallery';
+import UserMenu from '../UserMenu/UserMenu';
+import { Route, Link } from 'react-router-dom';
 
+export class App extends Component {
   async componentDidMount() {
-      const results = await getDefaultData()
-      await this.props.getAllMovies(results)
+    const results = await getDefaultData();
+    await this.props.getAllMovies(results);
   }
 
   render() {
     return (
-      <div>
-        <UserMenu />
-        {this.props.movies.length && <Gallery genre={'Now Playing'} data={this.props.movies[0]}/>}
-        {this.props.movies.length && <Gallery genre={'Popular'} data={this.props.movies[1]} />}
-        {this.props.movies.length && <Gallery genre={'Top Rated'} data={this.props.movies[2]} />}
-      </div>
-    )
+      <Route
+        exact
+        path='/'
+        render={() => (
```

Comment:
Nice job on the router and render, now we just gotta think about how we're going to setup a 404 page!